sdk/src/main/java/com/hedera/hashgraph/sdk/TransactionId.java

```diff
@@ -64,6 +63,13 @@ public final class TransactionId implements Comparable<TransactionId> {
     @Nullable
     private Integer nonce = null;
 
+    private static final long MICROSECONDS_PER_MILLISECOND = 1_000_000L;
```

Comment:
The name is not right, should be `NANOSECONDS_PER_MILLISECOND`.

Reply:
Please check #1605